src/main/java/backend/model/payment/SponsoringInvoice.kt

```diff
@@ -99,33 +103,36 @@ class SponsoringInvoice : Invoice {
     }
 
     override fun generatePurposeOfTransfer(): String {
-        TODO()
+        this.purposeOfTransferCode = generateRandomPurposeOfTransferCode()
+        this.purposeOfTransfer = "Spende $purposeOfTransferCode-BREAKOUT-${sponsor.lastname}"
+
+        return this.purposeOfTransfer!!
     }
 
     fun toEmailOverview(): String {
         return """
-        |<b>Challenges</b><br/>
+        |<b>Challenges</b>
         |${this.challenges.toEmailListing()}
-        |<br/><b>Sponsorings</b><br/>
-        |${this.sponsorings.toEmailListing()}<br/>
+        |<b>Sponsorings</b>
+        |${this.sponsorings.toEmailListing()}
         |
-        |<b>Total:</b> $amount<br/>
-        |<b>Already paid:</b> ${amountOfCurrentPayments()}<br/>
+        |<b>Total:</b> $amount
+        |<b>Already paid:</b> ${amountOfCurrentPayments()}
         """.trimMargin("|")
     }
 
     @JvmName("sponsoringToEmailListing")
     private fun List<Sponsoring>.toEmailListing(): String {
-        return this.map { it.toEmailListing() }.foldRight("") { acc, s -> "$acc<br/>\n$s" }
+        return this.map { it.toEmailListing() }.foldRight("") { acc, s -> "$acc\n$s" }
```

Comment:
The order indeed is important because otherwise we would have a leading `\n` iirc